config/routes.rb

```diff
@@ -643,7 +643,7 @@ def patch(*) end # Disable PATCH requests
       put "rebake"
       put "unhide"
       put "locked"
-      put "notice"
+      put "notice" => "posts#notice_action" # can’t use 'notice' with Rails 7.0.1. Might be a bug
```

Comment:
I’ve found this while I was searching for it: https://www.reddit.com/r/rails/comments/s8tiud/rails_7_are_notice_and_alert_actions_now_reserved/
I didn’t check again to see if it’s been resolved or not (since we are now at version 7.0.2.2 and not 7.0.1 anymore). Going to re-check.

Reply:
Just checked again and nothing changed on that matter :confused:

Reply:
Any official announcements that "notice" would become reserved?

Reply:
This breaking change was introduced in https://github.com/rails/rails/pull/42768 It effectively makes `notice` (and `flash`) a reserved names in the context of controllers.

Reply:
> Any official announcements that "notice" would become reserved?

So far I’ve found none.

Reply:
FYI here is a workaround - https://github.com/rails/rails/issues/44867#issuecomment-1095510173

Reply:
Nice, thanks! :slightly_smiling_face: